### DISCUSSION TITLE
What is the use of "_parse_sample(item)" in dataset.py?

### DISCUSSION THREAD

**Discussion by p0p4k** [coqui-ai/TTS#2469] [General Q&A]
https://github.com/coqui-ai/TTS/blob/624513018d8467e30f39bd67a658216dadd0a41a/TTS/tts/datasets/dataset.py#L21-L32
"wav_file" is always at index 1. Then why are we trying to parse it like that? Was it supposed to be a sanity check function at first then just forgotten about? Thanks.
ps. sorry for lots of questions today regarding the codes, i am trying to warp my head around all of this once in for all!

**Comment by erogol** [comment]
for matters return different items. parse_sample sets the arguments based on the list length. Probably we could find a smarter way.
